README.md

```diff
@@ -195,10 +195,17 @@ Add the following to your `config.php`:
                     2 => 'dbuser',
                     3 => 'dbuserpassword',
                     4 => 'xmppdomain',
+		    5 => true,
                 ),
             ),
     ),
 
+0 - Database Host
+1 - Prosody Database Name
+2 - Database User
+3 - Database User Password
+4 - XMPP Domain
+5 - Hashed Passwords in Database (true) / Plaintext Passwords in Database (false)
```

Comment:
I would add a insecure warning here for the plaintext option...

Reply:
Added.